src/doc/rustdoc/src/unstable-features.md

```diff
@@ -84,6 +84,37 @@ in documentation.
 `#![feature(doc_cfg)]` feature gate. For more information, see [its chapter in the Unstable
 Book][unstable-doc-cfg] and [its tracking issue][issue-doc-cfg].
 
```

Comment:
I think we should maybe put `doc_auto_cfg` in its own section (perhaps called `#![feature(doc_auto_cfg)]` or something like that). It's kind of a freestanding feature—i.e., you can use it without using `doc_cfg`.

Make sure to change "An extension to this feature" to "An extension to `doc_cfg`".